packages/app-builder-lib/src/targets/nsis/nsisOptions.ts

```diff
@@ -3,7 +3,7 @@ import { TargetSpecificOptions } from "../../core"
 
 export interface CustomNsisBinary {
```

Comment:
Wondering if it'd be worth extending resources interface here. What are your thoughts this? (just need to add back the jsdoc, but not sure how that works w.r.t. inherited properties... which could cancel out this idea)
```
export interface CustomNsisBinary extends CustomNsisResources {
   readonly debugLogging?: boolean | null
}
```

Reply:
I actually decided to go into other direction in the light of the comment below, and made all properties required and non-nullable.